openstack/blockstorage/extensions/quotasets/results.go

```diff
@@ -33,6 +33,66 @@ type QuotaSet struct {
 	BackupGigabytes int `json:"backup_gigabytes"`
 }
 
+// QuotaDetailSet represents details of both operational limits of block
+// storage resources and the current usage of those resources.
+type QuotaDetailSet struct {
+	// ID is the project ID associated with this QuotaDetailSet.
+	ID string `json:"id"`
+
+	// Volumes is the volume usage information for this project, including
+	// in_use, limit, reserved and allocated attributes. Note: allocated
+	// attribute is available only when nested quota is enabled.
+	Volumes QuotaDetail `json:"volumes"`
+
+	// Snapshots is the snapshot usage information for this project, including
+	// in_use, limit, reserved and allocated attributes. Note: allocated
+	// attribute is available only when nested quota is enabled.
+	Snapshots QuotaDetail `json:"snapshots"`
+
+	// Gigabytes is the size (GB) usage information of volumes and snapshots
+	// for this project, including in_use, limit, reserved and allocated
+	// attributes. Note: allocated attribute is available only when nested
+	// quota is enabled.
+	Gigabytes QuotaDetail `json:"gigabytes"`
+
+	// PerVolumeGigabytes is the size (GB) usage information for each volume,
+	// including in_use, limit, reserved and allocated attributes. Note:
+	// allocated attribute is available only when nested quota is enabled and
+	// only limit is meaningful here.
+	PerVolumeGigabytes QuotaDetail `json:"per_volume_gigabytes"`
+
+	// Backups is the backup usage information for this project, including
+	// in_use, limit, reserved and allocated attributes. Note: allocated
+	// attribute is available only when nested quota is enabled.
+	Backups QuotaDetail `json:"backups"`
+
+	// BackupGigabytes is the size (GB) usage information of backup for this
+	// project, including in_use, limit, reserved and allocated attributes.
+	// Note: allocated attribute is available only when nested quota is
+	// enabled.
+	BackupGigabytes QuotaDetail `json:"backup_gigabytes"`
+}
+
+// QuotaDetail is a set of details about a single operational limit that allows
+// for control of block storage usage.
+type QuotaDetail struct {
```

Comment:
And finally, this might work better as `QuotaUsage` since it's describing the usage of an individual quota item.

Again, I do realize that this also contains "limit" which isn't something "in use", but the other items describe the consumption/usage of the resource.